task-3/data-access/queries.sql

```diff
@@ -1,3 +1,4 @@
+-- Create users table
```

Comment:
Just a side note:
You might want to look into the topic of "migrations" since it seems that you're running these queries manually every time you spin up a new instance of your DB.
"knex" library is a popular solution for DB query management.